cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,42 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import ArticlePageObject from '../support/pages/article.pageObject';
+
+const articlePage = new ArticlePageObject();
+
 describe('Article', () => {
-  before(() => {
+  let article;
 
+  before(() => {
+    cy.task('generateArticle').then((generatedArticle) => {
+      article = generatedArticle;
+    });
   });
 
   beforeEach(() => {
     cy.task('db:clear');
   });
 
   it('should be created using New Article form', () => {
+    articlePage.visitNewArticlePage();
+    articlePage.createArticle(article.title, article.description, article.body);
 
+    cy.contains(article.title).should('be.visible');
   });
 
   it('should be edited using Edit button', () => {
+    articlePage.visitEditArticlePage(article.slug);
+    articlePage.editArticle('Updated Title',
+      'Updated Description', 'Updated Body');
 
+    cy.contains('Updated Title').should('be.visible');
   });
 
   it('should be deleted using Delete button', () => {
+    articlePage.visitArticlePage(article.slug);
```

Comment:
Verify that the `visitArticlePage` method is correctly implemented in the `ArticlePageObject` class. This method should navigate to the page of the article identified by `article.slug`.